eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -387,12 +393,30 @@ function GetExistingTags($apiUrl) {
   }
 }
 
-# Walk across all build artifacts, check them against the appropriate repository, return a list of tags/releases
-function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha,  $continueOnError = $false) {
-  $pkgList = [array]@()
+# Retrieve release tag for artiface package. If multiple packages, then output the first one.
+function RetrieveReleaseTag($pkgRepository, $artifactLocation, $continueOnError = $true) {
+  if (!$artifactLocation) {
+    return ""
+  }
+  try {
+    $pkgs, $ParsePkgInfoFn = RetrivePackages -pkgRepository $pkgRepository -artifactLocation $artifactLocation
+    if (!$pkgs -or !$pkgs[0]) {
+      return ""
```

Comment:
If there are no packages, should we throw?

Reply:
The change we have right now is better to have feature. If no package retrieved, then we don't do any replacement.

Reply:
The callers essentially need to handle if no release tag is found and do a fallback or error as needed there. In our case we essentially output a warning if there is now release tag found but we don't fail.